scenes/enumerated/problem_11946/problem_11946.py

```diff
@@ -0,0 +1,222 @@
+        brace_2 = BraceBetweenPoints(road_line_start_point, flag_point, direction=DOWN).shift(0.15 * UP)
+        brace_3 = BraceBetweenPoints(flag_point, road_line_end_point, direction=DOWN).shift(0.15 * UP)
+
+        self.play(AnimationGroup(
+            AnimationGroup(Write(brace_2), Write(brace_3)),
+                  lag_ratio=0.7))
+        self.wait()
+
+        # --------------------------------------------- Point 10 ------------------------------------- #
+        self.play(FadeOut(condition_1), FadeOut(condition_2_1), FadeOut(condition_2_2))
+        self.wait()
+
+        # --------------------------------------------- Point 11 ------------------------------------- #
+        road_second_part_text_percent = MathTex(text.ROAD_SECOND_PART_TEXT_PERCENT).move_to(road_second_part_center).shift(1.1 * UP).scale(1.2)
+        self.play(Write(road_second_part_text_percent))
+        self.wait()
+
+        # --------------------------------------------- Point 12 ------------------------------------- #
+        condition_3 = MathTex(text.CONDITION_3).next_to(task_number).scale(1.1).shift(0.6 * RIGHT + 0.05 * DOWN)
+        self.play(Write(condition_3), run_time=3)
+        self.wait()
+
+        # --------------------------------------------- Point 13 ------------------------------------- #
+        coord_x_t = ValueTracker(flag_point[0])
+        colored_line = always_redraw(lambda:Line(flag_point, [coord_x_t.get_value(), road_y_coord, 0], color=YELLOW))
+
+        self.play(FadeIn(colored_line))
+        self.play(
+            AnimationGroup(car_movement(car, road_second_part_70_percent),
+                           coord_x_t.animate(rate_func=rate_functions.linear).set_value(road_second_part_70_percent[0]),
+                           run_time=6))
+        self.wait()
+
+        # --------------------------------------------- Point 14 ------------------------------------- #
+        brace_4 = always_redraw(lambda:Brace(VGroup(dot, car)).shift(0.23 * UP + 0.065 * RIGHT))
+        self.play(ReplacementTransform(brace_3, brace_4), hour_14.animate.shift(1.2 * LEFT))
+        self.wait()
+
+        # --------------------------------------------- Point 15 ------------------------------------- #
+        self.play(FadeOut(flag_point_text))
+        self.wait()
+
+        # --------------------------------------------- New comments 2 ------------------------------------- #
+        dashed_line1 = DashedLine(dash_length=0.15, color=BLUE_D).set_length(3)
+        dashed_line2 = DashedLine(dash_length=0.15, color=BLUE_D).set_length(3)
+        dashed_line1.rotate(angle=-0.5 * PI, axis=OUT).move_to(flag_point).shift(1.7 * UP)
+        dashed_line2.rotate(angle=-0.5 * PI, axis=OUT).move_to(road_line_end_point).shift(1.7 * UP)
+        self.play(Create(dashed_line1), Create(dashed_line2), run_time=3)
+        self.wait()
+
+        # --------------------------------------------- New comments 3,4,5 ------------------------------------- #
```

Comment:
ինձ թվում ա ավելի հարմար կլինի (հիմնականում քեզ, բայց նաև մեզ), որ քոմմենթներն անելուց անիմացիան նկարագրես։ հետո կոդի մեջ բան գտնելը ավելի հեշտ կլինի էլի

Reply:
ինձ էսպեսա հարմար, քո ասած ձևով փորձել եմ, էտքան էլ հարմար չէր, բայց եթե տենցա պետք, տենց կանեմ

Reply:
իսկ եթե համ սենց կետերով գրես, որ քեզ հարմար լինի, համ էլ 1-2 բառով նաև գրես թե ինչ անիմացիա ա՞

Reply:
թեկուզ ոչ ամեն կետի հետ էլի, մի քանի կետը մեկ գրես, թե էդ բլոկում ինչ ա լինում ընդհանուր

Reply:
խոսքի # points 1-5  -  road, cars at the ends of the road, move the cars closer to each other until they meet

սենց մի բան էլի

Reply:
իսկ եթե անիմացիայի սկզբում գրեմ խնդիրը՞, նկատի չունեմ կետերը այ հենց խնդիրը

Reply:
ես էլ պուճուր բան ավելացնեմ, էլի
Էս կարող է խմբավորելու համար հաճելի է (ես էդ տիպի խմբավորման համար ընդամենը newline կդնեմ), բայց սովորական քոմմենթներն էլ են լավը, Point 19֊ը երևի դժվար է հիշել որ մասն էր, բայց եթե երկու բառ գրած էղնի, կռնաս դաժե search էնես։

Մեկ էլ, գոնե comment֊ի տողերի վրա 80 սիմվոլը չանցնինք, էլի, դաժե գիթհաբը չի կռնա մի տողի վրա ցույց տա (կարող է իմ էկրանի չափերից էլ է, բայց դե)։

Reply:
Անկեղծ ասած ես համաձայն չեմ որ նպատակահարմար կլինի, կարճ քոմմենթներ գրել, կետերը նշելու փոխարեն, համաձայն չեմ նաև որ դա կոգնի search, որտև ման էկողը ոնց պետքա իմանա թե ես ինչ եմ գրել։
Իմ կարծիքով իմասր չունի կարճ քոմենթներ մտածելու վրա ժամանակ ծախսել, չեմ ել կարծում որ դա օգնելույա, ինձ որ հաստատ չի օգնի կոդ գրելուց կամ փոփոխություներ անելուց, ես գիտհաբ իշուն բացում եմ point ը կարդում եմ, ու point ի համարով հեշտ գտնում եմ, կարծում եմ որպիսի օգնի քոմենթը ուրեմն ամբողջ point ի տեքստը պետքա գրել, դա էլ կնմնավի տեքստային դակումենտի ոչ թե կոդի, սկզբում փորձել եմ ձեր ասած ձևով անել, բայց տեսա որ իմ արծը ավելի հեշտա։
Արաջարկում եմ տվյալ folder ի մեջ մի հատ էլ .txt ֆայլ ավելացնել, ու մեջը հավաքել  առաջադրանքի տեքստը ու փոփոխություների տեկստը, բայց եթե որոշում կա որ բոլորս կոդը էտպես պետքա գրենք, այսինքն ամեն point ի համար համառոր քոմմենթ, իհարկե ես էլ կանեմ։

Reply:
քոմենթների սիմոլների քանակի մասով կարող եմ մակասացնել, բայց խնդիրը քո էկրաննա, կամ կարող ես zoom - անես, կսկսի տեղավորվել

Reply:
փոխարեն չեմ գրել
իրականում եթե լավ գրես, կիմանա ինչ ման գա (key word֊երով ման գա, ոչ թե ամբողջ քոմմենթը)

էդ ահագին է, երևի ավելորդ։ Իմ կողմից առաջարկ էր փոքր քոմմենթներին ավելի շատ նշանակություն տալ։ Ուրիշների մոտ էլ է էղել, որ գրել եմ։ Այլ դեպքերում կոնկրետ տողերի վրա սել եմ ինչի մասին արժե գրել։

Էս դեպքում ուղղակի կսեմ կռնա օգտակար էղնի փոքր ներկայացնել, եթե կարիք կա միտքը առանձնացնելու։ Եթե չես ուզե՝ մի էրա (եթե Նարեկը դեմ չէ)

Reply:
Հաջորդ վիդեոն կփորցեմ ձեր արաջարկածով անել